.github/workflows/build-package-docs.yaml

```diff
@@ -160,7 +150,7 @@ jobs:
       if: env.TARGET == 'production'
       env:
         BRANCH_NAME: ${{ github.event.inputs.ansible-package-version }}
-        PROD_URL: https://ansible.readthedocs.io/projects/ansible/
```

Comment:
FWIW this is not really important as there's a redirect on the RTD side that sends one elsewhere anyway.

Reply:
Ah, never mind. It's probably used somewhere internally…

Reply:
Marking the thread as resolved.